feature/home/src/main/java/org/sopt/official/feature/home/component/HomeToastButton.kt

```diff
@@ -0,0 +1,87 @@
+package org.sopt.official.feature.home.component
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import org.sopt.official.designsystem.Black
+import org.sopt.official.designsystem.MdsGray950
+import org.sopt.official.designsystem.Orange400
+import org.sopt.official.designsystem.Orange700
+import org.sopt.official.designsystem.SoptTheme
+import org.sopt.official.designsystem.White
+
+@Composable
+fun HomeToastButton(
+    longTitle: String,
+    missionDescription: String,
+    buttonText: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier
+            .fillMaxWidth()
+            .clip(CircleShape)
+            .background(Orange400)
+            .padding(13.dp),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Icon(
+            imageVector = ImageVector.vectorResource(id = org.sopt.official.feature.home.R.drawable.ic_soptamp_42),
+            tint = Color.Unspecified,
+            contentDescription = null,
+            modifier = Modifier.padding(end = 7.dp)
+        )
+
+        Column(
+            verticalArrangement = Arrangement.spacedBy(2.dp),
+            modifier = Modifier.padding(end = 4.dp)
```

Comment:
```suggestion
        Column(
            verticalArrangement = Arrangement.spacedBy(2.dp),
            modifier = Modifier.padding(end = 4.dp).weight(1f)
```
이렇게 하는건 어떤가요 이러면 아래쪽에 spacer 안써도 돼서...ㅎㅎㅎ

Reply:
근데 저보다 Spacer쓰는게 더 해석하기 편한거 같아요 저는